src/Main.java

```diff
@@ -77,9 +78,10 @@ public static void main(String[] args) {
         int thisYear = 2025;
         int lastYear = thisYear - 200;
         int nextYear = thisYear + 100;
-        while (lastYear < nextYear) {
-            System.out.println(lastYear);
-            lastYear += 79;
-        }
+        for (int i = 0; i < nextYear; i = i + 79)
```

Comment:
```suggestion
        for (int year = 0; year < nextYear; year = year + 79)
```
так понятнее было бы